### PR TITLE
adding message field to library_invite

### PR DESCRIPTION
**Pull request by ahsu1230**
message is an Option[String] and is a `text` in database

@yipingliao @andrewconner

### CODE REVIEW

**Comment by ahsu1230** [comment]
I actually right now have a message per invitee
But I see Asana has an overall message for all invites, changing it...

File: kifi-backend/modules/shoebox/app/com/keepit/controllers/website/LibraryController.scala [hunk 1]
```diff
@@ -188,13 +188,14 @@ class LibraryController @Inject() (
 
         val validInviteList = db.readOnlyMaster { implicit s =>
           invites.map { i =>
+            val message = (i \ "message").asOpt[String]
```
Comment (andrewconner) [inline kifi-backend/modules/shoebox/app/com/keepit/controllers/website/LibraryController.scala]: Mind making `""` be a `None` also?

**Comment by yipingliao** [comment]
Thanks! Is there a length limit on this string/what length should the front end limit this string to?

**Comment by ahsu1230** [comment]
http://help.scibit.com/mascon/masconMySQL_Field_Types.html

I think it says a normal `text` can have up to 16k characters should be enough right?

**Pull request merged**